### ISSUE TITLE
Question about flow rule.

### ISSUE THREAD

**Issue by beckend** [namib-project/nftables-rs#65]
```nftables
        flowtable flowed {
                hook ingress priority filter
                devices = { wan0, wg0 }
        }
        
         chain forward {
                type filter hook forward priority filter; policy accept;
                ct state established flow add @flowed
        }
```

How can I add `ct state established flow add @flowed`
I could not find any fitting API for it.

**Comment by jwhb** [comment]
@beckend thanks for reporting. I will have a look.

**Comment by jwhb** [comment]
FlowTable was already implemented, the Flow statement was not. This will be fixed by #67 and released as v0.5.0.

You'll find guidance on using the API for Flow/FlowTable in the [respective tests](https://github.com/namib-project/nftables-rs/pull/67/files#diff-0d1225d3617f2e9ed70638126f8164fefea7ce5c3689d438a7cd408d647b3841).

Your feedback is appreciated.
